dev-scheme/guile/guile1-1.8.8.recipe

```diff
@@ -0,0 +1,155 @@
+portVers="${portVersion%.*}"
+
+PROVIDES="
+	guile1$secondaryArchSuffix = $portVersionCompat
+	lib:libguile$secondaryArchSuffix = $libguileVersionCompat
+	lib:libguilereadline_v_17$secondaryArchSuffix = $libguileVersionCompat
+	lib:libguile_srfi_srfi_1_v_3$secondaryArchSuffix = 3.0.2
+	lib:libguile_srfi_srfi_13_14_v_3$secondaryArchSuffix = 3.0.1
+	lib:libguile_srfi_srfi_4_v_3$secondaryArchSuffix = 3.0.1
+	lib:libguile_srfi_srfi_60_v_2$secondaryArchSuffix = 2.0.2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
+	lib:libgc$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libunistring$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	guile1${secondaryArchSuffix}_tools$secondaryArchSuffix = $portVersionCompat
+	cmd:guile$commandSuffix = $portVersion
+	cmd:guile_snarf$commandSuffix = $portVersion
+	cmd:guile_tools$commandSuffix = $portVersion
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	guile1$secondaryArchSuffix == $portVersion base
+	lib:libgmp$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	guile1${secondaryArchSuffix}_devel = $portVersionCompat
+	cmd:guile_config$commandSuffix = $portVersion
+	devel:libguile$secondaryArchSuffix = $libguileVersionCompat
+	devel:libguilereadline_v_17$secondaryArchSuffix = $libguileVersionCompat
+	devel:libguile_srfi_srfi_1_v_3$secondaryArchSuffix = 3.0.2
+	devel:libguile_srfi_srfi_13_14_v_3$secondaryArchSuffix = 3.0.1
+	devel:libguile_srfi_srfi_4_v_3$secondaryArchSuffix = 3.0.1
+	devel:libguile_srfi_srfi_60_v_2$secondaryArchSuffix = 2.0.2
+	"
+REQUIRES_devel="
+	guile1$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libffi$secondaryArchSuffix
+	devel:libgc$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libunistring$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:gperf
+	cmd:grep
+	cmd:make
+	cmd:ld
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+defineDebugInfoPackage guile1$secondaryArchSuffix \
+	"$libDir"/libguile.so.$libguileVersion
+
+BUILD()
+{
+	CPPFLAGS="-D_BSD_SOURCE" LIBS="-lbsd -lnetwork" \
+	runConfigure --omit-dirs binDir ./configure \
+		--bindir=$commandBinDir \
+		--with-threads --disable-error-on-warning
+
+	# No MADV_DONTNEED, yet
+	sed --in-place '/HAVE_SYS_MMAN_H/d' config.h
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -f "$libDir"/charset.alias
+	rm -f "$libDir"/lib*.la
+	rm -f "$libDir"/guile/$portVers/extensions/*.la
+	rm -f "$libDir"/guile/$portVers/extensions/*.a
```

Comment:
@korli any thoughts on this? I know we don't like the *.la in the packages, but I the *.a files should be OK?

Reply:
Haven't checked ... but maybe the option "--disable-static" could resolve this?

Reply:
> @korli any thoughts on this? I know we don't like the *.la in the packages, but I the *.a files should be OK?

I copied this part from the recipe of Guile 2.2 . Where can I read about Haiku policies for libraries?  
Anyway I'm right now checking if --disable-static works (seem indeed that it does).

Reply:
Even with --disable-static added, the build process produces .la files. As far as I understand .la files are needed for libtool. Are they removed in Haiku?

Reply:
They are removed in Haiku yes, some recipes still have a mention "remove libtool file(s)"

Reply:
https://github.com/haikuports/haikuports/search?q=remove+libtool+file&unscoped_q=remove+libtool+file

Reply:
Static libraries aren't usually used on Haiku. But for ABI unstable projects they can still make sense.

Reply:
I think you can leave --disable-static out and use the remove of the *.la files, if there is a specific reason you want to remove the *.a files also I think we are good here :)

Reply:
Running a final build, install (with guile 2.2.4 installed) and check TeXmacs then

Reply:
Great! Thanks.

Reply:
Btw, I appreciate very much the carefulness you all put in this. Impressive.